worldengine/imex/__init__.py

```diff
@@ -12,24 +12,49 @@
 import sys
 
 
-def export(world, export_type, export_bpp, export_signed, export_normalize):
+def export(world, export_filetype, export_bpp, export_signed, path = 'seed_output', export_raw = False):
```

Comment:
Remove it? No way, you just renamed my export_normalize to export_raw. :)    Since we are giving the option to scale the data, thus normalize it, then the original name is more explanatory. That is only useful for image creation, which is why I had it in there to begin with.

I'm glad you are expanding it, I'm all for dropping Pillow and going full GDAL.